common/stats/disk_monitor.go

```diff
@@ -0,0 +1,97 @@
+package stats
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// DiskMonitor monitor disk usage for selected directories
+type DiskMonitor struct {
+	pathAbbrevs []string // these abbreviations are added to the end of the paths' usage gauge names
```

Comment:
think i'd prefer having paths & pathAbbrevs more closely linked than just relying on the indices to be correct. Could get confusing if we have a long list of paths. Maybe some pathAndAbbrev struct or even just a map[string]string

Reply:
Was going to suggest the same thing, prefer a slice of structs with all of the fields that can be iterated over.